--- beecrypt-4.2.1/configure.ac.orig	2009-09-18 17:22:53.408615127 +0200
+++ beecrypt-4.2.1/configure.ac	2009-09-18 17:43:48.912607585 +0200
@@ -456,7 +456,7 @@
       ac_cv_python_include="-I`$PYTHON -c 'import distutils.sysconfig; print distutils.sysconfig.get_python_inc()'`"
     ])
     AC_CACHE_CHECK([where to install python libraries], ac_cv_python_libdir, [
-      ac_cv_python_libdir=`$PYTHON -c 'import distutils.sysconfig; print distutils.sysconfig.get_python_lib()'`
+      ac_cv_python_libdir="${libdir}/python`$PYTHON -c 'import sys; print sys.version[[:3]]'`/site-packages"
     ])
   fi
 fi
