--- Makefile.am.orig	Tue Jun 10 22:42:33 2003
+++ Makefile.am	Thu Jun 26 17:40:42 2003
@@ -43,7 +43,7 @@
 
 BEECRYPT_OBJECTS = aes.lo aesopt.lo base64.lo beecrypt.lo blockmode.lo blockpad.lo blowfish.lo blowfishopt.lo dhaes.lo dldp.lo dlkp.lo dlpk.lo dlsvdp-dh.lo dsa.lo elgamal.lo endianness.lo entropy.lo fips186.lo hmac.lo hmacmd5.lo hmacsha1.lo hmacsha256.lo md5.lo memchunk.lo mp.lo mpopt.lo mpbarrett.lo mpnumber.lo mpprime.lo mtprng.lo rsa.lo rsakp.lo rsapk.lo sha1.lo sha1opt.lo sha256.lo timestamp.lo
 
-libaltdir=$(prefix)/lib@LIBALT@
+libaltdir=$(prefix)/lib
 
 libalt_LTLIBRARIES = libbeecrypt.la
 
