--- beecrypt-3.0.0pre/configure.ac.wiget	2003-06-10 23:32:46.000000000 +0200
+++ beecrypt-3.0.0pre/configure.ac	2003-06-11 00:34:19.000000000 +0200
@@ -649,7 +649,7 @@
     CPPFLAGS="$CPPFLAGS -DOPTIMIZE_ARM"
     ;;
   athlon*)
-    CPPFLAGS="$CPPFLAGS -DOPTIMIZE_I386 -DOPTIMIZE_MMX"
+    CPPFLAGS="$CPPFLAGS -DOPTIMIZE_I686 -DOPTIMIZE_MMX"
     ;;
   i386)
     CPPFLAGS="$CPPFLAGS -DOPTIMIZE_I386"
--- beecrypt-3.0.0pre/acinclude.m4.wiget	2003-06-10 23:32:34.000000000 +0200
+++ beecrypt-3.0.0pre/acinclude.m4	2003-06-11 00:33:24.000000000 +0200
@@ -288,7 +288,7 @@
     CFLAGS="$CFLAGS -O3 -fomit-frame-pointer"
     case $bc_target_cpu in
     athlon*)
-      CFLAGS="$CFLAGS -mcpu=pentiumpro";
+      CFLAGS="$CFLAGS -mcpu=$bc_target_cpu";
       ;;
     i586)
       CFLAGS="$CFLAGS -mcpu=pentium"
